### PR TITLE
OCTFD-173 #comment Implement design for imunify360 documentation

### PR DESCRIPTION
**Pull request by tastembekov**
Implementation of the new design

### CODE REVIEW

File: docs/.vuepress/public/arrow-down.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="9" viewBox="0 0 16 9">
```
Comment (telepenin) [inline docs/.vuepress/public/arrow-down.svg]: Why is it added into the project, this is not used directly

File: docs/.vuepress/public/arrow-up.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="9" viewBox="0 0 16 9">
```
Comment (telepenin) [inline docs/.vuepress/public/arrow-up.svg]: The same

File: docs/.vuepress/public/arrow-down.svg (hunk 1 above)
Comment (tastembekov) [reply docs/.vuepress/public/arrow-down.svg]: These are icons I used here https://monosnap.com/file/Ktp9XxizkkZEeYweNQM6KGGSjlQyNe. I put them there to have all assets in the same place. I'll move to the theme repo if it's not acceptable.

File: docs/.vuepress/public/arrow-up.svg (hunk 2 above)
Comment (tastembekov) [reply docs/.vuepress/public/arrow-up.svg]: The same

**Pull request merged**